pkg/virtctl/vm/vm.go

```diff
@@ -39,8 +40,8 @@ const (
 
 func NewStartCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "start (vm)",
-		Short:   "Start a virtual machine.",
+		Use:     "start vm",
```

Comment:
not sure about `vm` vs `(vm)`. E.g. `kubectl get --help` uses `NAME`... maybe we want to do the same?
/cc @rmohr

Reply:
I also see sometimes something like this in `kubectl`:

```
kubectl logs [-f] [-p] (POD | TYPE/NAME) [-c CONTAINER] [options]
```
I guess `(VM | NAME)`?

To be honest I am nost sure.

Reply:
mh, good point, but I think `(POD | TYPE/NAME)` means you can EITHER use the name of a pod (=`POD`) OR something like `deployment/nameOfDeployment` (=`TYPE/NAME`). So for us it would be just `VM` then, instead of my suggestion `NAME`.
Ah, too late, already merged ;)

Reply:
Ah yes. So `( NAME )` or `( VM )` then.